### ISSUE TITLE
Traing code error

### ISSUE THREAD

**Issue by amandaluof** [MooreThreads/Moore-AnimateAnyone#50]
Many thanks for releasing the training code.

However, when following the environment setting as well as data preparation and then running the command of stage 1 training, I got the error in the following screenshot. Is there anything wrong?

<img width="1681" alt="image" src="https://github.com/MooreThreads/Moore-AnimateAnyone/assets/44992004/5f898bf9-2822-4eb0-80a8-ad19c079a80a">

Looking forward to your reply. Thank you again !

**Comment by amandaluof** [comment]
This may come from gradient checkpointing.

**Comment by lixunsong** [comment]
Do you have gradient_checkpointing during stage 1 training? We turned it off, and perhaps some workarounds are needed here to avoid errors when gradient_checkpointing is enabled. We haven't investigated it thoroughly yet

**Comment by amandaluof** [comment]
> workarounds

Yes, I tried to turn on gradient_checkpointing. Thanks for your reply.
